content/200-concepts/100-components/01-prisma-schema/06-relations/400-self-relations.mdx

```diff
@@ -310,6 +310,7 @@ This relation expresses the following:
 - "a user can be followed by zero or more users"
 - "a user can follow zero or more users"
 
+TODO THis is only correct for relational databases, the MongoDB example is using lists and is Mongo's m:n - not really "implicit" (There is already a mention of this further down where it shows the SQL/data structure)
 Note that this many-to-many-relation is [implicit](many-to-many-relations#implicit-many-to-many-relations). This means Prisma maintains a [relation table](/concepts/components/prisma-schema/relations/many-to-many-relations#relation-tables) for it in the underlying database.
```

Comment:
Note for @keerlu

Reply:
Added that this note is for relational databases only